src/ResourceManager/SiteRecovery/Commands.SiteRecovery.Test/ScenarioTests/SiteRecoveryTests.ps1

```diff
@@ -359,4 +359,47 @@ function Test-SiteRecoveryVaultCRUDTests
 	Remove-AzureRmSiteRecoveryVault -Vault $vaultToBeRemoved
 	$vaults = Get-AzureRmSiteRecoveryVault -ResourceGroupName rg1 -Name v2
 	Assert-True { $vaults.Count -eq 0 }
-}
\ No newline at end of file
+}
+
+
+<#
+.SYNOPSIS
+Site Recovery Fabric Tests New model
+#>
+function Test-SiteRecoveryFabricTest
```

Comment:
updated the PR.

---
In reply to: [78818997](https://github.com/Azure/azure-powershell/pull/2912#discussion_r78818997) [](ancestors = 78818997,78799081)